reflect/src/main/java/dagger/reflect/ComponentInvocationHandler.java

```diff
@@ -75,6 +75,7 @@ private ComponentInvocationHandler(BindingGraph graph) {
             "Members injection methods may only return the injected type or void: " + method);
       }
 
+      @SuppressWarnings({"unchecked", "RedundantCast"})
```

Comment:
see https://youtrack.jetbrains.com/issue/IDEA-206560

Reply:
can you leave this as a comment in source?

Reply:
done

also fixed 2 dangling alignment issues like this:
https://github.com/JakeWharton/dagger-reflect/compare/9230378380e3d07a5ecbb3b3dad3d77b1e24b798..bf291c4220b14819da46e81f9e44c9fe9e9d823c#diff-6f0b84cc3cf4faa16145ef4af80a80daL41